doc/sphinx/proofs/writing-proofs/proof-mode.rst

```diff
@@ -528,31 +545,22 @@ Navigation in the proof tree
    .. exn:: No focused proof to restart.
       :undocumented:
 
-.. cmd:: Focus {? @natural }
-
-   Focuses the attention on the first goal to prove or, if :token:`natural` is
-   specified, the :token:`natural`\-th.  The
-   printing of the other goals is suspended until the focused goal
-   is solved or unfocused.
-
-   .. deprecated:: 8.8
-
-      Prefer the use of bullets or focusing brackets with a goal selector (see below).
-
-.. cmd:: Unfocus
-
-   This command restores to focus the goal that were suspended by the
-   last :cmd:`Focus` command.
+.. _focused_goals:
 
-   .. deprecated:: 8.8
-
-.. cmd:: Unfocused
+Focusing goals
+``````````````
 
-   Succeeds if the proof is fully unfocused, fails if there are some
-   goals out of focus.
+:gdef:`Focusing <focus>` lets you limit the context display to (initially) a
+single goal.  If a tactic creates additional goals from a focused goal, the
+subgoals are also focused.  The two focusing constructs are
+:ref:`curly braces <curly-braces>` (`{` and `}`) and :ref:`bullets <bullets>`
+(e.g. `-`, `+` or `*`).  These constructs can be nested.
 
 .. _curly-braces:
 
+Curly braces
```

Comment:
Curly braces and curly brackets are equivalent, right? I see that we are not completely consistent in the terminology. At least one error message and its description below use "brackets". Not a big deal though.

Reply:
The Internet says "The term "curly braces" is more favored in the U.S., while "brackets" is more widely used in British English.". I had never seen them referred to as "curly brackets".

Reply:
OK, let's try to consistently use "braces" everywhere in the docs and we can postpone fixing the error messages.

Reply:
So in US English it's curly braces and square brackets?

Reply:
In my experience, yes, but that may not be universal.  The Unicode table, which is probably meant to be American English, describes "{" as "left curly bracket".  The wikipedia article has the following chart, which has merely "braces".

![image](https://user-images.githubusercontent.com/1253341/193866849-f7fd9360-9e3e-42e3-b448-7127e71c1b23.png)

It also says "In most English-speaking countries, an unqualified word "bracket" refers to the parenthesis (round bracket); in the United States, the square bracket."

So I guess it should be "braces" or "curly brackets".

Reply:
I personally have never heard people use "curly bracket" in the UK.